client/js/parsers/edX/parser.spec.js

```diff
@@ -1,123 +1,39 @@
-        "status": 200,
-        "contentType": "application/xml",
-        "statusText": "OK",
-        "responseText": problem1
-      });
-
-      jasmine.Ajax.stubRequest('edXCourse/problem/d0ef2adedeba45038d69b24517892d1d.xml').andReturn({
-        "status": 200,
-        "contentType": "application/xml",
-        "statusText": "OK",
-        "responseText": problem2
-      });
-
-      jasmine.Ajax.stubRequest('edXCourse/problem/78934fbb26f44b2b85d252a4f3c52d54.xml').andReturn({
-        "status": 200,
-        "contentType": "application/xml",
-        "statusText": "OK",
-        "responseText": problem3
-      });
-
-      jasmine.Ajax.stubRequest('edXCourse/problem/d649f04c5979438fbe82334f07b7d6fe.xml').andReturn({
-        "status": 200,
-        "contentType": "application/xml",
-        "statusText": "OK",
-        "responseText": problem4
-      });
-
-      jasmine.Ajax.stubRequest('edXCourse/problem/8d6900d170f34deeb718866c2954c75f.xml').andReturn({
-        "status": 200,
-        "contentType": "application/xml",
-        "statusText": "OK",
-        "responseText": problem5
-      });
-
-      jasmine.Ajax.stubRequest('edXCourse/problem/da63a43c68024407aab8ca0f7c790b12.xml').andReturn({
-        "status": 200,
-        "contentType": "application/xml",
-        "statusText": "OK",
-        "responseText": problem6
-      });
-
-      jasmine.Ajax.stubRequest('edXCourse/problem/c34a20e2f1e24890baffcfc9ac68dcfc.xml').andReturn({
-        "status": 200,
-        "contentType": "application/xml",
-        "statusText": "OK",
-        "responseText": problem7
-      });
-
-      spyOn(AssessmentActions, 'edXLoadItem');
-
+      expectedHeaders = {
+        Accept: 'application/xml',
+      };
     });
 
     afterEach(() => {
-      jasmine.Ajax.uninstall();
+      nock.cleanAll();
     });
 
-    it('parses assessment xml from EdX into an object', () => {
-      settings = {
-        srcUrl: "edXCourse/sequential/97cc2d1812204294b5fcbb91a1157368.xml"
-      };
-
-      var assessment = Assessment.parseEdX(settings, $(sequential));
-
-      expect(assessment).toBeDefined();
-      expect(assessment.id).toEqual("sequential/97cc2d1812204294b5fcbb91a1157368");
-      expect(assessment.title).toEqual("Subsection One");
-      expect(assessment.standard).toEqual("edX");
-      expect(assessment.sections.length).toEqual(1);
-      expect(assessment.sections[0]).toEqual("04735103fe064c9da3a1a758bcda2692");
-
-      expect(jasmine.Ajax.requests.mostRecent().url).toBe('edXCourse/vertical/04735103fe064c9da3a1a758bcda2692.xml');
-      jasmine.Ajax.requests.mostRecent().respondWith({
-        "status": 200,
-        "contentType": "application/xml",
-        "statusText": "OK",
-        "responseText": vertical
-      });
-
-      expect(AssessmentActions.edXLoadItem).toHaveBeenCalled();
-
-    });
+    it('parses assessment xml from EdX into an object', () => {});
```

Comment:
hmm, it would be nice to add edx back in. Currently only the import is commented out, eventually we'd like to support it again. Maybe just comment out the test instead of deleting it, because testing the edx parser is something we will probably eventually want again.